.github/workflows/cifuzz.yml

```diff
@@ -0,0 +1,26 @@
+name: CIFuzz
```

Comment:
Can you please add the copyright / license at the top please?

Reply:
Took care of it.